TransportOperator/transportOperatorHT_class.f90

```diff
@@ -57,11 +57,15 @@ subroutine tracking_selection(self, p, tally, thisCycle, nextCycle)
     ! Get majornat XS inverse: 1/Sigma_majorant
     majorant_inv = ONE / self % xsData % getTrackingXS(p, p % matIdx(), MAJORANT_XS)
 
-    ! Obtain the local cross-section
-    sigmaT = self % xsData % getTrackMatXS(p, p % matIdx())
+    ! Obtain the local cross-section. Always choose ST in void
+    if (p % matIdx() == VOID_MAT) then
+      sigmaT = ZERO
+    else
+      sigmaT = self % xsData % getTrackMatXS(p, p % matIdx())
+    end if
```

Comment:
I'm amazed we hadn't seen this before. I'm curious whether this is the best place to put it rather than in the database? Putting it in the database might make for a tidier transportOperator, but might have some code duplication across different databases.

Reply:
It's not a common problem because it can only happen if particles are born in void, and ST and DT already had checks to make sure that this is ok. 
I think adding the check to the tracking operator rather than the database is ok. In the database the only thing you could do is set all the xss to zero, but that might give you troubles somewhere else.